### PR TITLE
Inbox total count unread

### PR DESCRIPTION
**Pull request by zofpolkowska**
Each IQ result stanza in conversation will provide total unread count and active conversation count

### CODE REVIEW

**Comment by mongoose-im** [comment]
[5432.1](https://travis-ci.org/esl/MongooseIM/jobs/419698181) / Erlang 19.3 / small_tests / cc2c2b6484b9b1361468d7b94cbf8f39b6579ef2
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2047/5432/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2047/5432/small_tests.19.3/small/ct_run.mongooseim@localhost.2018-08-23_15.36.24/lib.mongooseim.logs/run.2018-08-23_15.36.24/suite.log.html)

**Comment by mongoose-im** [comment]
[5436.5](https://travis-ci.org/esl/MongooseIM/jobs/420089229) / Erlang 19.3 / ldap_mnesia / 7f79c0582d5c0fc03f3c4fe0d758f88a60cfd5a8
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2047/5436/ldap_mnesia.19.3)/ [big](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2047/5436/ldap_mnesia.19.3/big/ct_run.test%40travis-job-8b6b7562-d6bb-46f0-9986-9d4afa7b1b87.2018-08-24_12.16.48/index.html)
**OK: 1034** / Failed: 0 / **User-skipped: 81** / Auto-skipped: 0

**Comment by mongoose-im** [comment]
[5438.1](https://travis-ci.org/esl/MongooseIM/jobs/420144737) / Erlang 19.3 / small_tests / 4bd772a42af1a80527a283906350194a2aefcdbc
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2047/5438/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2047/5438/small_tests.19.3/small/ct_run.mongooseim@localhost.2018-08-24_14.58.58/lib.mongooseim.logs/run.2018-08-24_14.58.58/suite.log.html)

**Comment by mongoose-im** [comment]
[5444.1](https://travis-ci.org/esl/MongooseIM/jobs/420985338) / Erlang 19.3 / small_tests / 466ee977c6cd235db78bb5f954f5e036533d9806
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2047/5444/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2047/5444/small_tests.19.3/small/ct_run.mongooseim@localhost.2018-08-27_07.59.18/lib.mongooseim.logs/run.2018-08-27_07.59.19/suite.log.html)

File: big_tests/tests/inbox_SUITE.erl [hunk 1]
```diff
@@ -482,6 +484,28 @@ reset_unread_counter_and_show_only_unread(Config) ->
...
                                                 end).
+check_total_unread_count_and_active_conv_count(Config) ->
+  escalus:story(Config, [{kate, 1}, {mike, 1}, {alice, 1}], fun(Kate, Mike, Alice) ->
+    % Kate sends message to Mike
+    MsgId =  <<"123123">>,
+    Msg1 = escalus_stanza:set_id(escalus_stanza:chat_to(Mike, <<"Hi mike">>), MsgId),
+    escalus:send(Kate, Msg1),
+    M1 = escalus:wait_for_stanza(Mike),
+    escalus:assert(is_chat_message, M1),
+    inbox_helper:get_inbox(Mike, #{ hidden_read => true }, 1),
```
[the remark this answers is not in the document]
Comment (zofpolkowska) [reply big_tests/tests/inbox_SUITE.erl]: Actually there's no need of calling `inbox_helper:get_inbox` here at all, I believe

**Comment by mongoose-im** [comment]
[5455.1](https://travis-ci.org/esl/MongooseIM/jobs/421143790) / Erlang 19.3 / small_tests / 92c4d6673bfb896176b4b767a12595cf77bd090e
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2047/5455/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2047/5455/small_tests.19.3/small/ct_run.mongooseim@localhost.2018-08-27_16.05.47/lib.mongooseim.logs/run.2018-08-27_16.05.48/suite.log.html)

**Comment by mongoose-im** [comment]
[5456.1](https://travis-ci.org/esl/MongooseIM/jobs/421442484) / Erlang 19.3 / small_tests / d62b4918f8c3213815c281216c5de3b4118776ea
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2047/5456/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2047/5456/small_tests.19.3/small/ct_run.mongooseim@localhost.2018-08-28_07.42.53/lib.mongooseim.logs/run.2018-08-28_07.42.53/suite.log.html)

**Comment by mongoose-im** [comment]
[5461.1](https://travis-ci.org/esl/MongooseIM/jobs/421464882) / Erlang 19.3 / small_tests / 33f5466034a3034aef27c2934bfa263de44bec73
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2047/5461/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2047/5461/small_tests.19.3/small/ct_run.mongooseim@localhost.2018-08-28_09.07.46/lib.mongooseim.logs/run.2018-08-28_09.07.47/suite.log.html)

**Pull request merged**